tensorflow/compiler/tf2tensorrt/convert/op_converter_registry.cc

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow/compiler/tf2tensorrt/convert/op_converter_registry.h"
+
+#include <set>
+#include <utility>
+
+#include "tensorflow/core/platform/mutex.h"
+
+#if GOOGLE_CUDA && GOOGLE_TENSORRT
+
+namespace tensorflow {
+namespace tensorrt {
+namespace convert {
+
+struct OpConverterRegistration {
+  OpConverter converter;
+  int priority;
+};
+class OpConverterRegistry::Impl {
+ public:
+  ~Impl() = default;
+
+  InitOnStartupMarker Register(const string& name, const int priority,
+                               OpConverter converter) {
+    mutex_lock lock(mu_);
+    auto item = registry_.find(name);
+    if (item != registry_.end()) {
+      const int existing_priority = item->second.priority;
+      if (priority < existing_priority) {
+        LOG(WARNING) << absl::StrCat(
+            "Ignoring custome TF->TRT op converter for op ", name,
+            " with given priority ", priority, " < ", existing_priority);
```

Comment:
We took the one with higher priority, that is "existing_priority". So here we should do
s/existing_priority/priority/
for the warning message. Right?

Reply:
It's saying "ignoring the user passed op converter function with priority (priority given by user) <= (existing_priority)"

So I should change to "<=". I'm not sure what you mean otherwise.

Reply:
revised to make as clear as possible